indico/modules/events/papers/util.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from indico.modules.events.contributions import Contribution
+from indico.modules.events.papers.models.revisions import PaperRevision
+
+
+def get_contributions_with_paper_submitted_by_user(event, user):
```

Comment:
Please make sure this query is not awfully slow for very big events with lots of contributions (e.g. CHEP2016)!